### PR TITLE
Convert tests to pytest fixtures

### PR DESCRIPTION
**Pull request by inkhey**
related to #619 
convert code to pytest fixtures

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: backend/tracim_backend/tests/library/test_workspace.py [hunk 1]
```diff
@@ -5,47 +5,50 @@
...
-from tracim_backend.models.data import Workspace
-from tracim_backend.tests import DefaultTest
-from tracim_backend.tests import eq_
-
+from tracim_backend.tests.fixtures import *  # noqa: F403,F40
+
+
+@pytest.mark.usefixtures("base_fixture")
+class TestThread(object):
+    # # FIXME - G.M - 2019-07-05 - TODO !!!
```
Comment (buxx) [inline backend/tracim_backend/tests/library/test_workspace.py]: You forgot something : )

**Comment by buxx** [comment]
Just do `FIXME - G.M - 2019-07-05 - TODO` and you can squash + merge.

File: backend/tracim_backend/tests/library/test_workspace.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/tests/library/test_workspace.py]: fixed

File: backend/tracim_backend/tests/fixtures.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    yield testing.setUp(settings=settings)
+    testing.tearDown()
+
+
+# @pytest.fixture
+# def depot():
+#     DepotManager._clear()
+#     DepotManager.configure("test", {"depot.backend": "depot.io.memory.MemoryFileStorage"})
+#     yield DepotManager.get("test")
+#     DepotManager._clear()
```
[the remark this answers is not in the document]
Comment (inkhey) [reply backend/tracim_backend/tests/fixtures.py]: deleted.

**Pull request merged**